WordPress/wporg-news-2021
# Add "Month in WordPress Title" block

Adds "Month in WordPress Title" block, which renders a heading (optionally with a link to the post). It outputs the short month name with an `aria-label` of "full month", and includes the year in the heading, so users browsing by heading have more context.

I also tightened up the spacing & font sizes to better match the design in figma.

**Screenshots**

1350px wide (this is the largest the text size gets)
![1350](https://user-images.githubusercontent.com/541093/153100060-8a2250f7-3f24-45fd-a17a-0f7357bcb570.png)

800px wide
![800](https://user-images.githubusercontent.com/541093/153100059-ee5644d2-180b-4a86-b0fc-50a428e711fc.png)

1350px wide (this is the smallest the text size gets)
![400](https://user-images.githubusercontent.com/541093/153100057-f5366885-3407-4f4f-94aa-917846a70a60.png)

**To test**

1. View the Month in WordPress category
2. It should match the design
3. Try navigating with a screen reader, the heading/links should be understandable, ex "March 2021"
